app/errors.py

```diff
@@ -0,0 +1,14 @@
+class NotWearingMaskError(BaseException):
```

Comment:
Inherit from class Exception are preferable if you create custom exception. 
`class Exception common base class for all non-exit exceptions`